pkg/minikube/registry/drvs/virtualbox/virtualbox.go

```diff
@@ -111,5 +111,15 @@ func status() registry.State {
 		}
 	}
 
-	return registry.State{Installed: true, Healthy: true}
+	version := ""
+	cmd = exec.CommandContext(ctx, path, "--version")
+	stdout, verr := cmd.Output()
+	if verr != nil {
+		klog.Warningf("unable to get virtualbox version, returned: %v", err)
+	} else {
+		version = string(stdout)
```

Comment:
how about we check the version and it is bellow certain version (older than 1 year) to tell the user about their outdated vbox using 
```
out.WarningT
```

Reply:
Hi @medyagh, Made the requested changes, please review